roles/confluent.ssl/tasks/export_certs_from_keystore_and_truststore.yml

```
@@ -31,20 +31,20 @@
     openssl pkcs12 -in {{keystore_path}} \
       -nodes -nocerts -out {{key_path}} \
       -passin pass:{{keystore_storepass}}
-  when: keytool_type.stdout == 'PKCS12'
+  when: keytool_type.stdout|lower == 'PKCS12'
```

```suggestion
  when: keytool_type.stdout|lower == 'pkcs12'
```

And I also think we should fix the one above: https://github.com/confluentinc/cp-ansible/blob/8daf3140882ddbe84cecf0320c52592374a1a66e/roles/confluent.ssl/tasks/export_certs_from_keystore_and_truststore.yml#L26